.gitignore

```diff
@@ -119,3 +119,7 @@ macos/pkg/Resources/*.lproj
 AdjustSdkDynamic.framework.zip
 AdjustSdkDynamic/
 AdjustSdk.framework/
+
+# ipmonitor
+tools/ipmonitor/ipmonitor
+tools/ipmonitor/ipmonitor.exe
```

Comment:
You got my hopes up for a Windows build

Reply:
In theory we can have it for windows too! Maybe after v2.5 we can update this code and implement it using the pcap for windows too.